src/mixins/object_geometry.mixin.js

```diff
@@ -341,10 +351,15 @@
         mtr: mtr
       };
 
-      // set coordinates of the draggable boxes in the corners used to scale/rotate the image
-      this._setCornerCoords && this._setCornerCoords();
+      if (normalized) {
+        this.oCoordsNormal = oCoords;
```

Comment:
One of the major changes I had to do was store normalized `oCoords` as well as group adjusted ones. Perhaps we don't need the group adjusted ones? I tried to take it out and it broke and so I didn't look too hard into it. But I could have been missing something.